gno.land/pkg/keyscli/call.go

```diff
@@ -74,6 +74,12 @@ func execMakeCall(cfg *MakeCallCfg, args []string, io commands.IO) error {
 	if cfg.PkgPath == "" {
 		return errors.New("pkgpath not specified")
 	}
+	if len(cfg.PkgPath) < 10 || cfg.PkgPath[:9] != "gno.land/" {
+		return errors.New("pkgpath should starts with gno.land/")
+	}
+	if len(cfg.PkgPath) >= 10 && (cfg.PkgPath[:10] == "gno.land/p" || cfg.PkgPath[:11] == "gno.land/p/") {
+		return errors.New("called to /p is forbidden")
+	}
 	if cfg.FuncName == "" {
```

Comment:
Can you just use the function `IsRealmPath`?

Also, this check should be primarily server-side. (`sdk/vm/keeper`)

Reply:
I moved the check to server side 👍